modules/recommended-etcd-practices.adoc

```diff
@@ -19,3 +19,32 @@ only accepts key reads and deletes. Some of the key metrics to monitor are
 database usage after a history compaction, and
 `etcd_debugging_mvcc_db_total_size_in_bytes` which shows the database size
 including free space waiting for defragmentation.
+
+Etcd writes data to disk, so its performance strongly depends on disk performance. Etcd 
+persists proposals on disk, slow disks and disk activity from other processes may cause 
```

Comment:
Might be better for the long sentence to be broken up.

Reply:
Fixed it, thanks.